### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by Kimpav** [schwabe/ics-openvpn#683]
Today, I get error on build step, cd main;./misc/build-native.sh
I am using android-ndk-r10e.
Here is the error message copy from my terminal:
```
[arm64-v8a] Compile        : openvpn <= compat-lz4.c
[arm64-v8a] Compile        : openvpn <= argv.c
[arm64-v8a] Compile        : openvpn <= base64.c
[arm64-v8a] Compile        : openvpn <= buffer.c
openvpn//src/openvpn/buffer.c: In function 'format_hex_ex':
openvpn//src/openvpn/buffer.c:472:5: error: 'for' loop initial declarations are only allowed in C99 or C11 mode
     for (int i = 0; i < size; ++i)
     ^
openvpn//src/openvpn/buffer.c:472:5: note: use option -std=c99, -std=gnu99, -std=c11 or -std=gnu11 to compile your code
make: *** [obj/local/arm64-v8a/objs/openvpn/src/openvpn/buffer.o] Error 1
make: *** Waiting for unfinished jobs....
Bradley@MacMini main (master) $
```

**Comment by schwabe** [comment]
NDK 10e is horrible outdated.

**Comment by kviktor** [comment]
Adding `-std=c11` to `APP_CFLAGS` seems to fix the issue (on v0.6.73).
